src/main/java/sparksoniq/jsoniq/runtime/iterator/functions/durations/DurationFunctionIterator.java

```diff
@@ -1,5 +1,6 @@
 package sparksoniq.jsoniq.runtime.iterator.functions.durations;
 
+import org.joda.time.Instant;
```

Comment:
A long-term comment: I read that Joda time was used before Java 8, that it was then largely taken over into Java 8, and that the developers of Joda time recommend migrating to the Java 8 library. This may be worth looking into later on (no urgency).

https://www.baeldung.com/joda-time